### PR TITLE
enable sentry performance tracing

### PR DESCRIPTION
**Pull request by hawkrives**

### CODE REVIEW

**Comment by hawkrives** [comment]
This enables https://docs.sentry.io/platforms/react-native/performance/

File: source/init/sentry.ts [hunk 1]
```diff
@@ -7,7 +7,17 @@ function install() {
...
 
-	Sentry.init({dsn: SENTRY_DSN})
+	Sentry.init({
+		dsn: SENTRY_DSN,
+
+		tracesSampleRate: 0.2,
+
+		integrations: [
+			new Sentry.ReactNativeTracing({
+				tracingOrigins: ['localhost', 'my-site-url.com', /^\//u],
```
Comment (drewvolz) [inline source/init/sentry.ts]: Do we need the my-site-url line?

**Review by drewvolz — APPROVED** [review]
Nice! One nonblocking comment

**Review by rye — APPROVED** [review]

File: source/init/sentry.ts (hunk 1 above)
Comment (hawkrives) [reply source/init/sentry.ts]: Oops, you are correct. I expect that we should change that to frogpond.tech…

**Pull request merged**